app/controllers/content_items_controller.rb

```diff
@@ -16,12 +18,17 @@ def show
   end
 
   def filter
-    @organisations = Organisation.all
+    @organisations = Organisation.order(:title)
+    @taxonomies = Taxonomy.order(:title)
   end
 
 private
 
   def set_organisation
     @organisation = Organisation.find_by(slug: params[:organisation_slug]) if params[:organisation_slug]
   end
+
+  def set_taxonomy
+    @taxonomy = Taxonomy.find_by(title: params[:taxonomy]) if params[:taxonomy]
```

Comment:
Addressed in https://github.com/alphagov/content-performance-manager/pull/95/commits/e5ede1820800f155f254f537dc4d17e94842d30d